superset/sqllab/command.py

```diff
@@ -132,6 +132,7 @@ def run(  # pylint: disable=too-many-statements,useless-suppression
                 ) from ex
             raise ex
         except Exception as ex:
+            logger.error("Query %d: %s", query.id, type(ex), exc_info=True)
```

Comment:
If we need the exception info in general it's better to use `logger.exception`, as it does that automatically:
```suggestion
            logger.exception("Query %d: %s", query.id, type(ex))
```